internal/core/integrations/gitlab_integration.go

```diff
@@ -556,6 +558,36 @@ func (g *gitlabIntegration) addProjectHook(ctx core.Context) error {
 
 }
 
+func CreateProjectHookOptions(token uuid.UUID, hooks []*gitlab.ProjectHook) (*gitlab.AddProjectHookOptions, error) {
```

Comment:
You do not need to export this function

```suggestion
func createProjectHookOptions(token uuid.UUID, hooks []*gitlab.ProjectHook) (*gitlab.AddProjectHookOptions, error) {
```